packages/frontend/src/pages/positions/PositionDetails.tsx

```diff
@@ -142,7 +142,7 @@ export function PositionDetails({
   account,
   pendingOffers,
 }: PositionDetailsProps) {
-  const ownedByYou = ethAddress === account
+  const ownedByYou = !!account && ethAddress === account.address
```

Comment:
```suggestion
  const ownedByYou = account?.address === ethAddress
```

Wouldn't this work as expected?